CHANGELOG.md

```diff
@@ -1,4 +1,66 @@
-# dbt_zuora version.version
+# dbt_zuora v0.3.2
+This release introduces the following updates.
+
+## Bug Fixes  
+- Flipped the variable logic in `zuora__using_multicurrency` in these models so that the `*_amount` values are called by default rather than `*_amount_home_currency` when the variable isn't set: ([#22](https://github.com/fivetran/dbt_zuora/pull/22))
+  - `zuora__line_item_history`: For setting the `gross_revenue` and `discount_revenue` values (these values are then used to calculate `net_revenue`).
+  - `zuora__monthly_recurring_revenue`: For setting the `mrr_expected_current_month` value.
+  - `int_zuora__transaction_grouped`: For setting daily amount values for invoices, discounts, taxes and credit balance adjustments to flow downsteram into the `zuora__account_daily_overview` model.
```

Comment:
This part is a lil confusing -- are these the affected fields for each end model? If so, I would phrase it more like "This affects..." rather than "For setting..."

Reply:
same for pre-release

Reply:
Updated.